pkg/vault/contracts/Vault.sol

```diff
@@ -1536,7 +1533,7 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
     }
 
     function _ensureValidTradeAmount(uint256 tradeAmount) private pure {
```

Comment:
This is probably not related to this PR, but maybe the comment over the _MINIMUM_TRADE_AMOUNT variable should be on top of this function as well, for clarity.